tests/fixtures/app/src/Controller/AuthorsController.php

```diff
@@ -18,15 +17,12 @@
  */
 class AuthorsController extends AbstractController
 {
-    private ContentManager $manager;
-
-    public function __construct(ContentManager $manager)
-    {
-        $this->manager = $manager;
-    }
-
     /**
-     * @Route("/{author}", name="author")
+     * @Route("/{author}", name="author", options={
+     *     "content": {
+     *         "dots": true,
```

Comment:
It does the job. But it feels very user unfriendly to me.
It adds complexity, make the user aware of an internal "problem" that should not get in their way.
I would love to find a way to get rid of it.

Maybe by relying on the response format / type rather than a file extention? I don't know.
Or maybe I'd rather have the user specify `"extention": "json"` on a file route (and leave `jane.doe` alone.

You know what I mean?